src/modules/recipe/lightDomQuery/lightDomQuery.js

```diff
@@ -0,0 +1,9 @@
+import { LightningElement } from 'lwc';
+
+export default class LightDomQuery extends LightningElement {
+    handleButtonClick() {
+        // Elements that are inside a Light DOM child component are directly accessible by the parent
+        this.template.querySelector('p.lightDomParagraph').innerText =
```

Comment:
I think this example is a bit confusing, as the name suggests that this is light DOM query, but you're reaching into a light DOM child.

Wouldn't it make more sense to switch `lightDomQuery` and `lightDomQueryChild`?

Reply:
Hmm, I don't know if we should switch the names, as the convention I followed for this and the other `styles` recipe is that of parent and child. The child component is the LightDOM component, and the parent is just a container. The name of the parent component is the one shown on the recipe card.

So should I call the parent components something like `lightDomQueryBasics`, and `lightDomStyleBasics`?

Reply:
I'm in line with the @adityanaag3 proposal since it will be visible in the app.